src/components/Button/Button.jsx

```diff
@@ -0,0 +1,8 @@
+export const Button = ({ name, btnClass }) => {
+  const btnClassName = "btn " + btnClass;
+  return (
+    <>
+      <button className={btnClassName}>{name}</button>
+    </>
+  );
+};
```

Comment:
I doubt that this component will respond to clicks. You need to pass props as below. This way the onClick that you design outside can be used here. Not completely sure but try and let me know

```JSX
export const Button = (props) => {
  const { name, btnClass } = props;
  const btnClassName = "btn " + btnClass;
  return (
    <>
      <button className={btnClassName} {...props}>{name}</button>
    </>
  );
};
```

Reply:
Thanks for pointing it out, refactored the code